README.md

```diff
@@ -177,6 +203,22 @@ Cache-Control: no-cache
 
 `````````````
 
+Possible Issues
+---------------
+
+You might experience very long startup times on some cloud hosted servers.  This might be because of a shortage
+of entropy as a result of no keyboard, or mouse:
+
+http://www.issihosts.com/haveged/
+
+To rectify, install the above software (Ubuntu instructions):
```

Comment:
If the build or startup process currently gets entropy from keyboard/mouse, instead of adding another package to work around the lack of those, can we specify that the process get entropy somewhere else?  Preferrably a source that's aleady available on the system, like: Java's `Random` class, `/dev/random` (or `/dev/urandom`), or [Advanced Tea Substitute](https://everything2.com/title/Advanced+tea+substitute)?